templates/docred/templates.yaml

```diff
@@ -91,3 +148,14 @@ templates:
     name: relation
     reference: ''
     task_template: true
+  a31dc527-a5b9-4411-a600-ea2bbe22a9d3: !Template
+    id: a31dc527-a5b9-4411-a600-ea2bbe22a9d3
+    jinja: "{%- set numbers = [] -%} \n{%- for ners in vertexSet -%}\n{%- for ner\
+      \ in ners if ner['type'] == 'NUM' -%}\n{{numbers.append(ner['name']) | default(\"\
+      \", True)}}\n{%- endfor -%}\n{%- endfor -%}\n{% if numbers %}\nFind all of the\
+      \ numbers in the text below. \n\n{% for sent in sents -%}\n{{ sent | join(\"\
```

Comment:
You might want to add "Please do not include years" or something, since a number will be denoted as a "time" if it's a year, e.g.


Lark Force was an Australian Army formation established in March 1941 during World War II for service in New Britain and New Ireland .Under the command of Lieutenant Colonel John Scanlan , it was raised in Australia and deployed to Rabaul and Kavieng , aboard SS Katoomba , MV Neptuna and HMAT Zealandia , to defend their strategically important harbours and airfields .The objective of the force , was to maintain a forward air observation line as long as possible and to make the enemy fight for this line rather than abandon it at the first threat as the force was considered too small to withstand any invasion .Most of Lark Force was captured by the Imperial Japanese Army after Rabaul and Kavieng were captured in January 1942 .The officers of Lark Force were transported to Japan , however the NCOs and men were unfortunately torpedoed by the USS Sturgeon while being transported aboard the Montevideo Maru .Only a handful of the Japanese crew were rescued , with none of the between 1,050 and 1,053 prisoners aboard surviving as they were still locked below deck .

The numbers are

1,050, 1,053.